test/extended/apiserver/patch.go

```diff
@@ -0,0 +1,91 @@
+package apiserver
+
+import (
+	"context"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	"github.com/openshift/origin/test/extended/apiserver/patch"
+	"github.com/openshift/origin/test/extended/testdata"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-api-machinery] JSON Patch", func() {
```

Comment:
Can you tag this with `operator.openshift.io` so that MicroShift will skip it?

Reply:
> Can you tag this with `operator.openshift.io` so that MicroShift will skip it?

still outstanding.